checkbox-provider-ce-oem/bin/tcp_multi_connections.py

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env python3
+import socket
+import argparse
+import threading
+import logging
+import time
+from datetime import datetime, timedelta
+
+logging.basicConfig(
+    level=logging.DEBUG,
+    format="%(asctime)s [%(levelname)s] %(message)s",
+    handlers=[
+        logging.StreamHandler(),
+    ],
+)
+
+
+class PortOutputer():
+    def __init__(self, port,
+                 message="",
+                 list_status={}):
+        self.port = port
+        self.message = message
+        self.list_status = list_status
+        self._status = None
+        self._total_period = None
+        self._avg_time_period = None
+        self._max_time_period = None
+        self._min_time_period = None
+
+    @property
+    def status(self):
+        if not self.list_status:
+            return "ERROR"
```

Comment:
we could define the status enumeration in this file, and report corresponding result.
```
from enum import Enum


class StatusEnum(Enum):
    SUCCESS = 0
    ERROR = 1
    FAIL = 2

    def status(self):
        if not self.list_status:
            return StatusEnum.ERROR
```